blog/content/introtogonum.md

```diff
@@ -0,0 +1,155 @@
+# The Gonum Numeric Computing Packages
+
+## Gonum -- The Basics
+
+Gonum is a set of packages designed to make writing numeric and scientific
+algorithms productive, performant, and scalable.
+
+Gonum is a set of packages written in the 
+[Go programming language](https://www.golang.org) (more on Go later), and is
+not a language on its own.
+This makes Gonum similar to [numpy](https://www.numpy.org) and
+[scipy](https://www.scipy.org), libraries built on top of
+[python](https://www.python.org), and different from
+[Julia](https://julialang.org/) and [Matlab](https://www.mathworks.com/)
+which are full programming languages.
+
+Gonum contains libraries for [matrices and linear algebra](https://godoc.org/gonum.org/v1/gonum/mat);
+[statistics](https://godoc.org/gonum.org/v1/gonum/stat), 
+[probability](https://godoc.org/gonum.org/v1/gonum/stat/distuv) 
+[distributions](https://godoc.org/gonum.org/v1/gonum/stat/distmv), 
+and [sampling](https://godoc.org/gonum.org/v1/gonum/stat/sampleuv); tools for
+[function differentiation](https://godoc.org/gonum.org/v1/gonum/diff/fd), 
+[integration](https://godoc.org/gonum.org/v1/gonum/integrate/quad),
+and [optimization](https://godoc.org/gonum.org/v1/gonum/optimize);
+[network](https://godoc.org/gonum.org/v1/gonum/graph) creation and analysis; and more.
+
+## Gonum Philosophy
+The Gonum packages are designed to be simple, efficient and composable.
+There is a tension, especially in scientific computing, between ease and simplicity.
+Gonum, [like the Go language itself](https://commandcenter.blogspot.com/2012/06/less-is-exponentially-more.html),
+aims first to provide reliable and predictable code, while trying to maintain
+ease-of-use.
+Gonum is different from other popular scientific libraries, in that the Go language
+does not allow operator or method overloading.
+On the surface, this often means that the same algorithm requires a few more
+lines of code in Go.
+We fully recognize that sexy one-liners are rare with Gonum.
+
+We accept this cost because we believe it comes with an enormous benefit.
+The Gonum function and method signatures make the algorithmic outputs clear,
+and it is easy to identify the code being executed and examine the implementation.
+Science is built on understanding how an experiment was performed and
+what the results are.
+We believe scientific code should be no different.
+In fact, Gonum is partly distinguished from other libraries in that the entire
+codebase can be run with pure Go -- you only need to know one programming
```

Comment:
Are you distinguishing from other libraries for Go or other libraries in general. I would assume that it is other libraries for Go since otherwise the claim is trivially true, but how many other scientific libraries are there for Go?